cmd/k6build/main.go

```diff
@@ -10,6 +10,7 @@ import (
 func main() {
 	root := newRootCmd()
 	root.AddCommand(newVersionCmd())
+	root.AddCommand(newBuildCmd())
```

Comment:
I don't think it's worth introducing subcommands for these simple CLI tools because of the version command. If build itself is the root command, version can be a flag.
I mean, these CLI tools provided with the library typically consist of a command, which will later become a sub-command in k6 (or xk6).

Reply:
I think it is premature to know this. But agree we should keep this tool simple.